### ISSUE TITLE
Add UDIM example to unit tests

### ISSUE THREAD

**Issue by ld-kerley** [AcademySoftwareFoundation/MaterialX#2007]
There is a small example linked [here](https://github.com/AcademySoftwareFoundation/MaterialX/pull/1381#issuecomment-1595859977). 

I think it would be good to include this, or some other similarly small example,  directly in the repo, for use in unit tests.

**Comment by jstone-lucasfilm** [comment]
Yes, a multi-UDIM example material would be a great improvement, if anyone has the bandwidth to contribute one!

**Comment by Arin1604** [comment]
Hi sorry for the late request, I would love to work on this! 

I'm very new to open source software but this sounds like a very interesting project!

**Comment by jstone-lucasfilm** [comment]
Welcome @Arin1604, and it's all yours!

**Comment by jstone-lucasfilm** [comment]
@Arin1604 has let us know that school commitments will prevent him from tackling this project, so I'll mark this as free for others to work on.  Thanks for taking a look, Arin, and feel free to join other ASWF Dev Days events in the future!

**Comment by jstone-lucasfilm** [comment]
Thanks to @ld-kerley for this original report, as well as for the fix in #2113!
